# Review Sheet for Quiz II - DNA Barcoding and PCR

## Vocabulary terms may include:

* DNA Barcoding- classification method that uses a short genetic marker in an organism's DNA to identify it as belonging to a particular species
* DNA Barcoding region (locus)
* PCR-Polymerase Chain Reaction: Allows us to selectively amplify/copy specific DNA sequences 
* Primer- shorter pieces of DNA used in PCR to help extract our info9 sorta like a search bar put some info in to get some)
* DNA Polymerase- The DNA polymerases are enzymes that create DNA molecules by assembling nucleotides, the building blocks of DNA
* Taq Polymerase- a thermostable DNA polymerase
=======
* DNA Barcoding- a taxonomic method that uses a short genetic marker in an organism's DNA to identify it as belonging to a particular species.
* DNA Barcoding region (locus)- the specific location or position of a gene, DNA sequence, on a chromosome.
* PCR- Polymerase Chain Reaction
* Primer- Oligo, a strand of short nucleic acid sequences (generally about 10 base pairs) that serves as a starting point for DNA synthesis. It is required for DNA replication because the enzymes that catalyze this process, DNA polymerases, can only add new nucleotides to an existing strand of DNA.
* DNA Polymerase- Enzymes that create DNA molecules by assembling nucleotides, the building blocks of DNA.
* Taq Polymerase- thermostable DNA polymerase, used in PCR. 
* Supernatant- The liquid lying above a solid residue after crystallization, precipitation, centrifugation, or other process.


### Contributions from your notes
PCR is used to  amplify a DNA target sequence. The PCR primers are short pieces of DNA that will get the reaction started. You have to at least put in the 4 base pairs to get an answer. The probability of A in a sequence of 1 billion base pairs is 1/4 and the probability of A being next to a T is 1/16... it's all exponential. In order to use PCR, you have to know how the target starts and ends and then you can decide what primers to use. DNA polymerase(enzyme) makes more copies of DNA by filling in the complementary strand. PCR has three stages using DNA polymerase. Stage one is the melting point at about 95 Celsius. During this stage, a double strand of DNA becomes a single strand of DNA. Next is the annealing stage at about 50 Celsius. During this stage, the reaction is cooled down just enough for the primers to bind. The last stage is the extension stage. During this stage, the bonds formed by the primers are now short double stranded DNA. •	How much resolution? How much clarity/detail?
Ex- A butterfly. If we wanted to classify butterflies, since there is lots of diversity, it needs lots of resolution. When things are similar, we need lots of resolution. (Like a monarch and a viceroy butterfly). 
•	Conserved genes don’t have lots of differences. 
•	In the cell, there is DNA. There are genes, and non genes. 
•	There is an interesting piece of DNA, its called a target sequence. We want to amplify it, know more about it. 
•	PCR Primer- short pieces of DNA (aka an oligio, it means short). 
•	We can’t retrieve information without putting information into the system. 
Ex- Google ignores filler words/prepositions. When searching for DNA you can only search ACTG. 
•	A primer is like a word, we won’t get results for just “a”, we need a longer combination. 
•	A human genome is 3 billion bp. 
•	A primer cannot be just one nucleotide, it has to be more. 
•	Specious- something plausible and that sounds true, but it’s actually false. 
•	Chargaff’s Ratio- In each species, the ration of ATCG is specific. 
•	The probability of an “a” in a genome is ¼. The probability of a “t” is ¼. The probability of “A” and “T” is 1/16. 
•	So every 16 nucleotides, expect an AT match. 
•	Sequence that is too long will only have 1/billion match. You just keep multiplying. It’s exponential. 
•	You need one key piece of PCR for it to work. We need to know the start and the end for the first and last few nucleotides
•	So we can design primers that match where it starts and ends. 
•	So once you know how story starts and ends, you know what you’re looking for. 
•	At a certain point in a reaction, have a single stranded DNA, and will add an enzyme present in our reaction. 
•	Enzyme is called DNA Polymerase. It makes a copy of DNA, and it fill in the complimentary strand. 
•	Only one thing the enzyme needs to get started, the primer. 
•	How do we create the situation?
•	Stages of PCR:
1.	Melting Stage- At 95 c. The double stranded DNA becomes single stranded DNA. Gets so hot it becomes single stranded. Hydrogen bonds break. 
2.	Annealing- 50 c. The primers bind, we cool down the reaction. Big strands of DNA are cooling down, but not enough to come together and band, but the oligos are small, so the primers cool down quickly enough that they will bind. Cooled to annealing temperature is cool enough so only the primers bind to what they’re looking for. 
3.	Extension- At 72 c- The primers don’t fall off yet. Double stranded DNA now is only where we have matches. So polymerase will only bind there and replicate. 
4.	The first time, we end up with something incomplete. A piece of DNA that starts but never completes. And the copy ends but never starts. In the next round there’s a new match, it completes the DNA, and gets a piece that matches. But the time you get to the third, you get short pieces that are amplified matched to target sequences and quick, predominate in a reaction. You end up with over 3 billion copies per match. 


## Concepts and questions that should be tested include:

* Resolution - How do gene/non-gene regions of DNA help us resolve between species and non-species
Genes are highly conserved regions of DNA from species to species while non-genes are often not conserved. Therefore when looking at genetic barcode we look at the genetic part of the DNA. This allows us to examine the genes, and the higher genomic DNA % conserved the more closely related they are.
* How does PCR work (reagents/steps/etc.)?
* 3 Stages of PCR:
* 1. Melting Stage - occurs at 95 degrees celsius, this is when the douvle stranded DNA becomes single stranded DNA
* 2. Annealing Stage - occurs at 50 degrees celsius, at this point the PCR Primers bind. The reaction is cooled so that the biggers strands of DNA dont have enough time to bond but since the PCR Primers are small (oligos) they cool down faster and bind to their match
* 3. Extension - occurs at 72 degrees celsius, the double stranded DNA have matches and the primers are still attached so the polymerase will bend there and replicate those portions of DNA

* How does DNA extraction work(reagents/steps/etc.)? 
Steps of DNA Extraction:
1.	Grind plant tissue with nuceli lysis buffer
2.	Incubate sample
3.	Add RNAse
4.	Add protein precipitation solution. Mix gently, incubate. 
5.	Centrifuge at max for three minutes. 
6.	Transfer supernattent, 
7.	Add isoproponal 
8.	Put in fridge
9.	Centrifuge at max speed for one minute
10.	Pour off supernatant and add ethanol. 
11.	Centrifuge at max speed for one minute and pipette off ethanol. 
12.	Add rehydration solution. 


### Contributions from your notes

## Calculations to know

* How to calculate the probability of a PCR primer of length n matching in a genome of length l
Humans have about 3 billion base pares so therefore we need to put in the least amount of base pairs that will still allow our answer to only come up once i.e. out of three billion have around 750 million or 1/4 "A"s and another 1/4 "T"S but if we were to look for "AT" we would multiply the 2 fractions and get 1/16 that means we would expect it to show up once every 16 time we continue to multiply by 1/4 (meaning that our #s will be growing exponentially) until we reach 3 billion.

### Contributions from your notes


## Other useful resources

* [Video on PCR (2D)](https://www.dnalc.org/resources/animations/pcr.html)
* [Video on PCR (3D)](https://www.dnalc.org/resources/3d/19-polymerase-chain-reaction.html)
* [DNA Barcoding Animation](https://www.dnalc.org/resources/animations/dna-barcoding.html)
* [DNA Replication ](https://www.dnalc.org/resources/3d/04-mechanism-of-replication-advanced.html)(not covered in class but useful)

* PCR Explained http://learn.genetics.utah.edu/content/labs/pcr/
* DNA Barcoding Explained http://www.barcodeoflife.org/content/about/what-dna-barcoding

* [PCR] http://study.com/academy/lesson/taq-polymerase-definition-function-quiz.html

https://www.youtube.com/watch?v=iQsu3Kz9NYo 
This is a helpful PCR simulation showing how PCR works and the different steps of it enacted in a way that is easy to understand.

## Suggested Quiz Questions (class contributed)
1. Why is the isopropanol replaced by ethanol?
2. Humans have around 3 billion base pairs in their DNA about how long would PCR primer have to be?

 "A'" 1/4 * "T" 1/4 = 1/16 "AT" 16* "AT" 1/16 =1/32 ............8 bases 1/1024... 16 bases 1/1048576.....24 bases 1/107373741824  25 bases 1/ 4 billion so between 24 to 25 bases 
2. What is the step of adding ethanol called and why?
>The wash because it washes away all excess carbs and proteins making the sample as pure as possible. 
>Because ethanol is a lighter molecular weight alcohol and it takes less energy and evaporates more quickly
4. Why is there a higher advantage of getting the exact targeted space by repeating the extension step more than once?
5.  How much resolution is needed to compare a monarch and viceroy butterfly? Explain.
(maybe insert a picture of them both: http://www.gpnc.org/images/jpegs/animals/monarchf.jpg 
                                      https://networkofideas.files.wordpress.com/2010/11/monarch-butterfly1.jpg )


