# Review Sheet for Quiz II - DNA Barcoding and PCR

## Vocabulary terms may include:
* DNA Barcoding-
* DNA Barcoding region (locus)
* PCR-Polymerase Chain Reaction: Allows us to selectivly ampifly/copy specific DNA sequences 
* Primer- shorter peices of DNA used in PCR to help extract our info9 sorta like a search bar put some info in to get some)
* DNA Polymerase
* Taq Polymerase

### Contributions from your notes
PCR is used to  amplify a DNA target sequence. The PCR primers are short pieces of DNA that will get the reaction started. You have to atleast put in the 4 base pairs to get an answer. The probability ot A in a sequence of 1 billion base pairs is 1/4 and the probability of A being next to a T is 1/16... it's all exponential. In order to use PCR, you have to know how the target starts and ends and then you can decide what primers to use. DNA polymerase(enzyme) makes more copies of DNA by filling in the complementary strand. PCR has three stages using DNA polymerase. Stage one is the melting point at about 95 celcius. During this stage, a double strand of DNA becomes a single strand of DNA. Next is the annealing stage at about 50 celcius. During this stage, the reaction is cooled down just enough for the primers to bind. The last stage is the extention stage. During this stage, the bonds formed by the primers are now short double stranded DNA. 

## Concepts and questions that should be tested include:

* Resolution - How do gene/non-gene regions of DNA help us resolve between species and non-species
Genes are highly conserved regions of DNA from speiceis to species while non-genes are often not conserved. Therefore when looking at genetic barcode we look at the genetic part of the DNA. This allows us to examine the genes, and the higher genomic DNA % conseved the more closley related they are.
* How does PCR work (reagents/steps/etc.)?

* How does DNA extraction work(reagents/steps/etc.)? 


### Contributions from your notes

## Calculations to know

* How to calculate the probability of a PCR primer of length n matching in a genome of length l
Humans have about 3 billion base pares so therefore we need to put in the least amout of base pairs that will still allow our answer to only come up once i.e. out of three billion have around 750 million or 1/4 "A"s and another 1/4 "T"S but if we were to look for "AT" we would multipy the 2 fractions and get 1/16 that means we would expect it to show up once every 16 time we contiune to mitiply by 1/4 (meaning that our #s will be growing exponetaily) until we reach 3 billion.

### Contributions from your notes


## Other useful resources

* [Video on PCR (2D)](https://www.dnalc.org/resources/animations/pcr.html)
* [Video on PCR (3D)](https://www.dnalc.org/resources/3d/19-polymerase-chain-reaction.html)
* [DNA Barcoding Animation](https://www.dnalc.org/resources/animations/dna-barcoding.html)
* [DNA Replication ](https://www.dnalc.org/resources/3d/04-mechanism-of-replication-advanced.html)(not covered in class but useful)
* PCR Explained http://learn.genetics.utah.edu/content/labs/pcr/
* DNA Barcoding Explained http://www.barcodeoflife.org/content/about/what-dna-barcoding


## Suggested Quiz Questions (class contributed)
1. Why is the isopropanol replaced by ethanol?
2. Humans have around 3 billion base pairs in their DNA about how long would PCR primer have to be?
A. "A'" 1/4 * "T" 1/4 = 1/16 "AT" 16* "AT" 1/16 =1/32 ............8 bases 1/1024... 16 bases 1/1048576.....24 bases 1/107373741824  25 bases 1/ 4 billion so between 24 to 25 bases 
